src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.io.File;
+
+public interface FileReaderService {
+    String read(File file);
```

Comment:
your method should get `String` parameter, not `File`

Reply:
done